sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -231,14 +230,13 @@ void gnrc_rpl_parent_update(gnrc_rpl_dodag_t *dodag, gnrc_rpl_parent_t *parent)
     if (parent != NULL) {
         parent->lifetime = (now / SEC_IN_USEC) + ((dodag->default_lifetime * dodag->lifetime_unit));
         if (parent == dodag->parents) {
-            ipv6_addr_t all_RPL_nodes = GNRC_RPL_ALL_NODES_ADDR;
             kernel_pid_t if_id;
-            if ((if_id = gnrc_ipv6_netif_find_by_addr(NULL, &all_RPL_nodes)) != KERNEL_PID_UNDEF) {
+            if ((if_id = gnrc_ipv6_netif_find_by_addr(NULL, &ipv6_addr_all_rpl_nodes)) != KERNEL_PID_UNDEF) {
                 fib_add_entry(&gnrc_ipv6_fib_table,
                               if_id,
                               (uint8_t *) ipv6_addr_unspecified.u8,
                               sizeof(ipv6_addr_t),
-                              (FIB_FLAG_NET_PREFIX | 0x0),
+                              FIB_FLAG_NET_PREFIX,
```

Comment:
`(FIB_FLAG_NET_PREFIX | 0x0)` is just a more verbose version of `FIB_FLAG_NET_PREFIX`